controller/state/statedb_dml.go

```diff
@@ -132,4 +132,12 @@ const (
 		UPDATE tasks SET worked_by = $2
 		WHERE uuid = $1
 	`
+
+	deleteTaskSQL = `
+		DELETE FROM tasks WHERE uuid = $1
+	`
+
+	deleteTaskStatusLedgerSQL = `
+		DELETE FROM task_status_ledger WHERE uuid = $1
```

Comment:
It might make sense to combine these into a single query:
```
deleteTaskAndStatueSQL = `
    DELETE FROM tasks WHERE uuid = $1;
    DELETE FROM task_status_ledger WHERE uuid = $1
`
```

Reply:
oh did I not push the fix? you actually can great ride of this with ON DELETE CASCADE